CSS/CSS.sublime-syntax

```diff
@@ -7,6 +7,14 @@ file_extensions:
   - css.erb
 scope: source.css
 variables:
+  # Many variable names taken directly from https://www.w3.org/TR/CSS22/grammar.html#scanner
+  h: '[0-9a-fA-F]'
+  unicode: '\\{{h}}{1,6}(?:\r\n|[ \t\r\n\f])?'
+  escape: '(?:{{unicode}}|\\[^\r\n\f0-9a-fA-F])'
+  nmstart: '(?:[_a-zA-Z]|[^[:ascii:]]|{{escape}})'
+  nmchar: '(?:[_a-zA-Z0-9-]|[^[:ascii:]]|{{escape}})'
+  ident: '-?{{nmstart}}{{nmchar}}*'
+
```

Comment:
Regarding `nmstart` & `nmchar`: By only matching alphabetic/alphanumeric characters, a large portion of the character table is unusable even though the spec allows it. Was this your intention?

I tested about 20 characters from various non-alphabetic ranges in the latest version of Chrome and all were parsed correctly as the first character of a classname, except for the space characters.